slick/src/main/scala/slick/util/ConstArray.scala

```diff
@@ -420,7 +421,7 @@ object ConstArray {
     new ConstArray(a)
   }
 
-  def from[T](values: Traversable[T]): ConstArray[T] = {
+  def from[T](values: Iterable[T]): ConstArray[T] = {
```

Comment:
To keep this binary compatible you must keep using `Traversable` (deprecated type alias for Iterable in 2.13)